sites/fast-website/src/public/index.html

```diff
@@ -131,7 +131,160 @@ <h3>AngularJS (1.x) 1.7.9</h3>
         <!-- documentation:start -->
         <section class="section">
           <div class="section-component">
-            Documentation components
+            <card-section>
+							<feature-card>
```

Comment:
Something off with the major intentation of `<feature-card>`.  Can we pull that back in?  seem to also happen on properties of `<fast-anchor>` also?  If Prettier is doing this we should correct.

Reply:
Prettier does not target html files currently.

Reply:
Yeah, I noticed that.  It doesn't reflect the file in my local, so I suspect it is a prettier issue.
<img width="1003" alt="Screen Shot 2020-05-14 at 11 32 31 AM" src="https://user-images.githubusercontent.com/26206206/81971919-b0349f00-95d6-11ea-96b0-f5bd92ad1b6c.png">

Reply:
> Prettier does not target html files currently.

Is it something I can't see in my html file that I could fix?  Or a quirk of GitHub perhaps?

Reply:
Not a simple way to fix this currently with any kind of script. I'd just drop the formatting all the way back and try to re-indent and see if it's fixed. It could be an issue with your editor settings; hard to tell without digging in.

Reply:
I think I got it.  My editor was setting tabs instead of spaces to indent on return even though I don't have it set to do so as far as I could find.